### PR TITLE
Resolved issues with edit access modal

### PR DESCRIPTION
**Pull request by sreenug**
Added CSS for check box
Added a new event to clear embargo dates on close
Added on blur option to ignore clearing of embargo date

### CODE REVIEW

**Comment by daines** [comment]
An image, "loading.gif" is referenced by `jqueryui-editable.css`, but it isn't anywhere in our project. This is causing a 404 error when the "Add embargo" link is clicked.

**Comment by daines** [comment]
If an embargo is chosen by clicking "Add embargo", then selecting a date, and then clicking "Update", no embargo is actually set.

File: access/src/main/external/static/css/admin/jquery-ui.css [hunk 1]
```diff
@@ -955,6 +955,7 @@ body .ui-tooltip {
 
 .ui-icon-close { background-position: -80px -128px; }
 .ui-icon-closethick { background-position: -96px -128px; }
+.ui-icon-check { background-position: -64px -144px; }
```
Comment (daines) [inline access/src/main/external/static/css/admin/jquery-ui.css]: Is this a patch to this version of jquery-ui, or has the CSS been updated to be in sync with the jquery-ui JavaScript?

Comment (sreenug) [inline access/src/main/external/static/css/admin/jquery-ui.css]: This patch is just for this version of Jquery. CSS of Jquery Editable library has not been updated.

**Comment by sreenug** [comment]
Resolved the issues discovered...

File: access/src/main/external/static/js/admin/src/EditAccessControlForm.js [hunk 2]
```diff
@@ -31,6 +32,13 @@ define('EditAccessControlForm', [ 'jquery', 'jquery-ui', 'ModalLoadingOverlay',
 				}
 				var formattedDate = moment(params.newValue).format('YYYY-MM-DD[T]HH:mm:ss');
 				self.addAttribute(self.accessControlModel, 'embargo-until', formattedDate, self.aclNS, self.aclPrefix);
+			}).on('hidden', function(e, reason) {
+				if(reason === 'cancel') {
+					$(".add_embargo", this.element).editable('setValue', null);
+			        self.removeAttribute(self.accessControlModel, 'embargo-until', self.aclPrefix);
+					return;
```
Comment (bbpennel) [inline access/src/main/external/static/js/admin/src/EditAccessControlForm.js]: For consistency, please switch space indents to tabs in this block.

**Comment by bbpennel** [comment]
loading.gif was added twice, at:

access/src/main/external/static/css/images/loading.gif
access/src/main/external/static/css/admin/images/loading.gif

It is only referenced in the admin UI, can we get rid of the other one?

File: access/src/main/external/static/js/admin/src/EditAccessControlForm.js [hunk 3]
```diff
@@ -31,6 +32,13 @@ define('EditAccessControlForm', [ 'jquery', 'jquery-ui', 'ModalLoadingOverlay',
 				}
 				var formattedDate = moment(params.newValue).format('YYYY-MM-DD[T]HH:mm:ss');
 				self.addAttribute(self.accessControlModel, 'embargo-until', formattedDate, self.aclNS, self.aclPrefix);
+			}).on('hidden', function(e, reason) {
+				if(reason === 'cancel') {
```
Comment (daines) [inline access/src/main/external/static/js/admin/src/EditAccessControlForm.js]: Please use a space between "if" and "("

File: access/src/main/external/static/js/admin/src/EditAccessControlForm.js (hunk 2 above)
Comment (sreenug) [inline access/src/main/external/static/js/admin/src/EditAccessControlForm.js]: Done.

File: access/src/main/external/static/js/admin/src/EditAccessControlForm.js (hunk 3 above)
Comment (sreenug) [inline access/src/main/external/static/js/admin/src/EditAccessControlForm.js]: Done

**Comment by bbpennel** [comment]
Looks good, thank you Sreenu!

**Pull request merged**